File: api/src/main/java/com/sdrfood/api/entity/Garcom.java
package com.sdrfood.api.entity;

import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;
import javax.persistence.Table;

@Entity
@Table(name = "tb_garcom")
public class Garcom {
	
	@Id
	@GeneratedValue(strategy = GenerationType.AUTO)
	private Integer id;
	
	private String nome;
	private String dt_nasc;
	private String fone;
	private String rua;
	private String bairro;
	private String numero;
	private String cpf;
	private String escolaridade;
	
	public Garcom() {
		
	}
	
	public Garcom(Integer id, String nome, String dt_nasc, String fone, String rua, String bairro, String numero,
			String cpf, String escolaridade) {
		super();
		this.id = id;
		this.nome = nome;
		this.dt_nasc = dt_nasc;
		this.fone = fone;
		this.rua = rua;
		this.bairro = bairro;
		this.numero = numero;
		this.cpf = cpf;
		this.escolaridade = escolaridade;
	}

	public Integer getId() {
		return id;
	}

	public void setId(Integer id) {
		this.id = id;
	}

	public String getNome() {
		return nome;
	}

	public void setNome(String nome) {
		this.nome = nome;
	}

	public String getDt_nasc() {
		return dt_nasc;
	}

	public void setDt_nasc(String dt_nasc) {
		this.dt_nasc = dt_nasc;
	}

	public String getFone() {
		return fone;
	}

	public void setFone(String fone) {
		this.fone = fone;
	}

	public String getRua() {
		return rua;
	}

	public void setRua(String rua) {
		this.rua = rua;
	}

	public String getBairro() {
		return bairro;
	}

	public void setBairro(String bairro) {
		this.bairro = bairro;
	}

	public String getNumero() {
		return numero;
	}

	public void setNumero(String numero) {
		this.numero = numero;
	}

	public String getCpf() {
		return cpf;
	}

	public void setCpf(String cpf) {
		this.cpf = cpf;
	}

	public String getEscolaridade() {
		return escolaridade;
	}

	public void setEscolaridade(String escolaridade) {
		this.escolaridade = escolaridade;
	}

}

File: APISDRFood/Sources/Model/Garcom.php
<?php

class Garcom {

    private $nome;
    private $dt_nasc;
    private $fone;
    private $rua;
    private $bairro;
    private $num;
    private $cpf;
    private $escolaridade;

    /**
     * Get the value of nome
     */ 
    public function getNome()
    {
        return $this->nome;
    }

    /**
     * Set the value of nome
     *
     * @return  self
     */ 
    public function setNome($nome)
    {
        $this->nome = $nome;

        return $this;
    }

    /**
     * Get the value of dt_nasc
     */ 
    public function getDt_nasc()
    {
        return $this->dt_nasc;
    }

    /**
     * Set the value of dt_nasc
     *
     * @return  self
     */ 
    public function setDt_nasc($dt_nasc)
    {
        $this->dt_nasc = $dt_nasc;

        return $this;
    }

    /**
     * Get the value of fone
     */ 
    public function getFone()
    {
        return $this->fone;
    }

    /**
     * Set the value of fone
     *
     * @return  self
     */ 
    public function setFone($fone)
    {
        $this->fone = $fone;

        return $this;
    }

    /**
     * Get the value of rua
     */ 
    public function getRua()
    {
        return $this->rua;
    }

    /**
     * Set the value of rua
     *
     * @return  self
     */ 
    public function setRua($rua)
    {
        $this->rua = $rua;

        return $this;
    }

    /**
     * Get the value of bairro
     */ 
    public function getBairro()
    {
        return $this->bairro;
    }

    /**
     * Set the value of bairro
     *
     * @return  self
     */ 
    public function setBairro($bairro)
    {
        $this->bairro = $bairro;

        return $this;
    }

    /**
     * Get the value of num
     */ 
    public function getNum()
    {
        return $this->num;
    }

    /**
     * Set the value of num
     *
     * @return  self
     */ 
    public function setNum($num)
    {
        $this->num = $num;

        return $this;
    }

    /**
     * Get the value of cpf
     */ 
    public function getCpf()
    {
        return $this->cpf;
    }

    /**
     * Set the value of cpf
     *
     * @return  self
     */ 
    public function setCpf($cpf)
    {
        $this->cpf = $cpf;

        return $this;
    }

    /**
     * Get the value of escolaridade
     */ 
    public function getEscolaridade()
    {
        return $this->escolaridade;
    }

    /**
     * Set the value of escolaridade
     *
     * @return  self
     */ 
    public function setEscolaridade($escolaridade)
    {
        $this->escolaridade = $escolaridade;

        return $this;
    }

}